openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java

```diff
@@ -449,6 +452,26 @@ public static <U> List<U> getEntitiesFromSeedData(String entityType, String path
     return entities;
   }
 
+  /**
+   * Merge the contents from the json files in the path into a single JsonObject
+   */
+  public static JsonObject getBindingsFromSeedData(String path) throws IOException {
+    JsonObjectBuilder jsonObjectBuilder = Json.createObjectBuilder();
```

Comment:
@pmbrull what is the need for this

Reply:
I'll remove it and use another approach suggested by mohit